cmd/gofish/list.go

```diff
@@ -54,7 +74,13 @@ func findFood() []string {
 				continue
 			}
 			if len(files) > 0 {
-				fudz = append(fudz, f.Name())
+				fileName := f.Name()
+				rigConf, err := ioutil.ReadFile(filepath.Join(barrelPath, fileName) + "/rig.conf")
```

Comment:
I'm not sure why you called `filepath.Join`, and then appended `/rig.conf` at the end. Because of that change, this won't work on Windows.